src/main.rs

```diff
@@ -55,6 +55,9 @@ fn main() -> PassResult<()> {
         }
     }
 
+    opts.lib_path
+        .push(option_env!("CALYX_PRIMITIVES_DIR").unwrap_or(".").into());
```

Comment:
This will always add the primitives dir to the lib path which is not the behavior of the existing set up.